[2025-09-17T17:45:00.791+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-17T17:45:00.853+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-17T15:30:00+00:00 [queued]>
[2025-09-17T17:45:00.866+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-17T15:30:00+00:00 [queued]>
[2025-09-17T17:45:00.867+0200] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-17T17:45:00.890+0200] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_customers> on 2025-09-17 15:30:00+00:00
[2025-09-17T17:45:00.902+0200] {standard_task_runner.py:64} INFO - Started process 1502 to run task
[2025-09-17T17:45:00.910+0200] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_api_to_bi', 'extract_customers', 'scheduled__2025-09-17T15:30:00+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/etl_api_to_bi.py', '--cfg-path', '/tmp/tmpiti6gg6r']
[2025-09-17T17:45:00.918+0200] {standard_task_runner.py:91} INFO - Job 102: Subtask extract_customers
[2025-09-17T17:45:01.147+0200] {task_command.py:426} INFO - Running <TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-17T15:30:00+00:00 [running]> on host 1621268d5064
[2025-09-17T17:45:01.998+0200] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data' AIRFLOW_CTX_DAG_ID='etl_api_to_bi' AIRFLOW_CTX_TASK_ID='extract_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-09-17T15:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-17T15:30:00+00:00'
[2025-09-17T17:45:02.011+0200] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-17T17:45:02.189+0200] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-09-17T17:45:02.190+0200] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2025-09-17T17:45:02.250+0200] {python.py:237} INFO - Done. Returned value was: 1
[2025-09-17T17:45:02.255+0200] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-17T17:45:02.323+0200] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_api_to_bi, task_id=extract_customers, run_id=scheduled__2025-09-17T15:30:00+00:00, execution_date=20250917T153000, start_date=20250917T154500, end_date=20250917T154502
[2025-09-17T17:45:02.395+0200] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-17T17:45:02.449+0200] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-17T17:45:02.452+0200] {local_task_job_runner.py:222} INFO - ::endgroup::
